contracts/migration/localhost/POR/20230831181923_deploy.json

```diff
@@ -0,0 +1,17 @@
+{
+  "deploy": {
+    "name": "e-Gold",
+    "timeout": 0,
+    "validator": "0x0000000000000000000000000000000000000000",
+    "decimals": 8,
```

Comment:
@bayram98 Have you discussed this decimals number with Bryan or e-gold?

Reply:
Decimals not discussed it yet, I will put on agenda.
 
Current, deployment configuration is as a default settings. We will need to double check it later.